test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -337,6 +337,80 @@ public void HubsCannotHaveOverloadedMethods()
         }
 
         [Fact]
+        public async Task CannotCallStaticHubMethods()
+        {
+            var serviceProvider = CreateServiceProvider();
```

Comment:
I thought the tests would be more unit-testy..  For instance the original bug is about discovering methods defined on `Object` but there is no test testing this. This can be easily tested without doing clients etc. just by invoking HubReflectionHelper and verifying what method it discovered.